src/System.Text.Json.Dynamic/System/Text/Json/JsonDynamicObject.cs

```diff
@@ -323,6 +334,9 @@ public bool TryFormat(Span<byte> buffer, out int written, TextFormat format, Enc
 
     static class Utf8StringExtensions
     {
+        private static readonly char[] EncodingChars = new char[] { '"' };
+        private static readonly ReadOnlySpan<char> QuoteSpan = new ReadOnlySpan<char>(EncodingChars, 0, 1);
+
```

Comment:
It was in a different class. None of this code will stay long-term. This change is a move towards getting rid of an undesirable method.